File: Practice/14/C++/14/14/14.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
	long long n;
	cin >> n;
	if (n < 0) exit(0);
	if (n == 0) { cout << 0; exit(0); }
	int sum = 0;
	int x = 1;
	double Log;
	while(x <= n)
	{
		Log = log2(x);
		if (Log == int(Log)) sum++;
		x++;
	}
	cout << sum;
}

File: Practice/14/Python/14.py
n = int(input())
import sys
if n > 10**15 or n < 0: sys.exit()
if n == 0: print (0); sys.exit()
from math import log2
sum = 0
x = 1
Log = 0
while x <= n:
    Log = log2(x)
    if Log == int(Log):
        sum += 1
    x += 1
print(sum)
